gbmodel/model_sqlalchemy.py

```diff
@@ -23,6 +23,19 @@ def get_professors(self, id):
             return False
         return result
 
+    # Checks if professor ID exists in the DB
```

Comment:
I'm not going to bother blocking anything for this until cleanup week but this and other comments function/class descriptions should go in a docstring. See PEP 257 for some good examples.